src/django_components/expression.py

```diff
@@ -72,6 +109,24 @@ def is_aggregate_key(key: str) -> bool:
     return ":" in key and not key.startswith(":")
 
 
+def is_spread_operator(value: Any) -> bool:
+    if not isinstance(value, str) or not value:
+        return False
+
+    return value.startswith("...")
```

Comment:
There needs to be something after the spread operator too right?

Reply:
I didn't check before, but added a check now, [here](https://github.com/EmilStenstrom/django-components/pull/596/files#diff-7345171ed38e1dd2e4b8199b65a2fd082323bdb331ff0ef0dc0dd3bd43f111aeR422). So if it's just `...` without a value, it raises an error